extensions/csrc/cuda/utils/vec_type_traits.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
 
-#include <c10/macros/Macros.h>
```

Comment:
You might want to remove `#include <c10/macros/Macros.h>` in `extensions/csrc/cuda/utils/vec_copy.h` as well.

Reply:
Done